pkg/operator/resources/operator/operator.go

```diff
@@ -976,23 +1050,126 @@ the disk alias on ovirt DiskMappings is respected only when provided in context
 	}
 }
 
+func createOperatorDeployment(operatorVersion, namespace, deployClusterResources, operatorImage, controllerImage, pullPolicy string) *appsv1.Deployment {
+	deployment := CreateOperatorDeployment("vm-import-operator", namespace, "name", "vm-import-operator", serviceAccountName, int32(1))
```

Comment:
"vm-import-operator" can be replaced with constant operatorName

Reply:
ack